### DISCUSSION TITLE
Enablement of Github secret scanning

### DISCUSSION THREAD

**Discussion by vmuskaaan** [community/community#33034] [Code Security] [answered]
Is there anybody out there you can help me understand pricing model associated with Github Secret scanning ? We are already enterprise user. 
We want to solve for Scanning secrets on github when a PR is raised or so. We are comparing the pricing model. Any leads would be helpful.
TIA!

**Comment by jhutchings1** [comment, marked as answer]
Secret scanning is a feature of GitHub Advanced Security. Our sales team can walk you through the cost and benefits of this product suite. https://github.com/enterprise/contact?utm_source=github&utm_medium=site&utm_campaign=adv-security&ref_page=/features/security&ref_cta=Contact%20Sales&ref_loc=hero
